### DISCUSSION TITLE
How to adjust the width of selectbox column?

### DISCUSSION THREAD

**Discussion by peifengsi** [KevinVandy/material-react-table#432] [Q&A]
Hi,
When the tableLayout is set to "fixed", the selectbox column has too much space. I'd like to find a way to shrink it. Thanks in advance.

![image](https://user-images.githubusercontent.com/39690374/225854530-3c77d87f-4451-450c-86ef-e9629ddd5080.png)

**Comment by peifengsi** [comment]
BTW, just figured out that the below code can adjust the width of selectbox column, but seems the minimal "size" is around 50. How can I set a smaller one?

`
          displayColumnDefOptions={{
            'mrt-row-select': {
              enableColumnActions: true,
              enableHiding: true,
              size: 50,
            },
          }}
`

**Reply by belshaa** [reply]
Try this:
layoutMode='grid'
displayColumnDefOptions={{ 'mrt-row-select': { size: 30, minSize: 30, maxSize: 30 }, }}

**Reply by peifengsi** [reply]
> Try this: layoutMode='grid' displayColumnDefOptions={{ 'mrt-row-select': { size: 30, minSize: 30, maxSize: 30 }, }}

Thanks. It does shrink the width of selectbox column, but the total table width will be larger than the window width because of some columns have much content to fill in. I used the "fixed layout" to avoid scrolling the table horizontally.
